runtime/plugins/m3_aggregator.go

```diff
@@ -62,25 +62,26 @@ func InitializeM3Collector(scope tally.Scope) *M3CollectorClient {
 }
 
 // NewM3Collector creates a collector for a specific circuit. The
-// prefix given to this circuit will be {circuit_name}.circuitbreaker.{metric}...
+// prefix given to this circuit will be circuitbreaker.{metric}...
 // Circuits with "/" in their names will have them replaced with ".".
 func (m *M3CollectorClient) NewM3Collector(name string) metricCollector.MetricCollector {
 	name = strings.Replace(name, "/", "-", -1)
 	name = strings.Replace(name, ":", "-", -1)
 	name = strings.Replace(name, ".", "-", -1)
+	tagMap := map[string]string{"emitter-name": name}
 	return &M3Collector{
-		scope:                   m.scope,
-		attemptsPrefix:          name + ".circuitbreaker" + ".attempts",
-		errorsPrefix:            name + ".circuitbreaker" + ".errors",
-		successesPrefix:         name + ".circuitbreaker" + ".successes",
-		failuresPrefix:          name + ".circuitbreaker" + ".failures",
-		rejectsPrefix:           name + ".circuitbreaker" + ".rejects",
-		shortCircuitsPrefix:     name + ".circuitbreaker" + ".shortCircuits",
-		timeoutsPrefix:          name + ".circuitbreaker" + ".timeouts",
-		fallbackSuccessesPrefix: name + ".circuitbreaker" + ".fallbackSuccesses",
-		fallbackFailuresPrefix:  name + ".circuitbreaker" + ".fallbackFailures",
-		totalDurationPrefix:     name + ".circuitbreaker" + ".totalDuration",
-		runDurationPrefix:       name + ".circuitbreaker" + ".runDuration",
+		scope:                   m.scope.Tagged(tagMap),
+		attemptsPrefix:          "circuitbreaker" + ".attempts",
```

Comment:
seems like string concatenation is no necessary, should just be "circuitbreaker.attempts". Same below.